sdk/src/main/java/com/ifountain/opsgenie/client/model/beans/Restriction.java

```diff
@@ -1,15 +1,13 @@
 package com.ifountain.opsgenie.client.model.beans;
 
-import com.ifountain.opsgenie.client.model.beans.ScheduleRotationRestriction.DAY;
-
 /**
  * Restriction bean
  */
 public class Restriction extends Bean {
     private Integer endHour;
-    private Integer endMinute;
+    private Integer endMin;
     private Integer startHour;
-    private Integer startMinute;
+    private Integer startMin;
```

Comment:
Are we getting StartMin or EndMin for response. If yes, Can we add @JsonProperty and keep name startMinute and endMinute which looks correct name